src/Controller/AdminRegistrationController.php

```diff
@@ -124,6 +139,13 @@ public function finishRegistration(HTTPRequest $request): HTTPResponse
      */
     public function removeRegisteredMethod(HTTPRequest $request): HTTPResponse
     {
+        if (!$request->isDELETE()) {
+            return $this->jsonResponse(
+                ['errors' => [_t(__CLASS__ . '.WRONG_METHOD', 'Wrong HTTP request method used')]],
+                400
```

Comment:
```suggestion
                405
```

Reply:
Since this can't be reached via any other HTTP method now, is it even necessary to keep this check in place?

Reply:
Yep good point!